src/components/WeekDayPicker/weekDay.js

```diff
@@ -21,6 +21,8 @@ export default function WeekDay(props) {
     const weekDayId = useUniqueIdentifier('week-day');
     const inputType = multiple ? 'checkbox' : 'radio';
 
+    const [isFocused, setIsFocused] = useState();
```

Comment:
If you want to pass styles when focused you don't need to use state, simply make a selector that takes the label that is a sibling of the input focused